beacon-chain/rpc/nodev1/node.go

```diff
@@ -45,8 +45,14 @@ func (ns *Server) GetVersion(ctx context.Context, _ *ptypes.Empty) (*ethpb.Versi
 
 // GetSyncStatus requests the beacon node to describe if it's currently syncing or not, and
 // if it is, what block it is up to.
-func (ns *Server) GetSyncStatus(ctx context.Context, _ *ptypes.Empty) (*ethpb.SyncingResponse, error) {
-	return nil, errors.New("unimplemented")
+func (ns *Server) GetSyncStatus(_ context.Context, _ *ptypes.Empty) (*ethpb.SyncingResponse, error) {
+	headSlot := ns.SyncChecker.HeadSlot()
```

Comment:
Another simplified approach is to instantiate the following in `Server`:
```go
HeadFetcher blockchain.HeadFetcher
```
Then you can do:
```go
headSlot := ns.HeadFetcher.HeadSlot()
```

It seems anti-pattern to implement `HeadSlot()` for `initial-sync` service as the relayer to pass head slot. The required services should be getting this info directly from `block-chain` service

```
// what you have today:
rpc service -> initial sync service -> block chain service
// we should be able to simplify it to:
rpc service -> block chain service
```

Reply:
Yes, I definitely should change it. Thanks!